### ISSUE TITLE
#Preview in 'QRCodeView.swift' is not recognized 

### ISSUE THREAD

**Issue by QBlockQ** [reez/Monday#56]
Library 'QRCodeView.swift' has a bug where the build failed due to #Preview in code line 78 as they're not using the correct syntax for SwiftUI previews.  We have fixed it by using the below:

struct QRCodeView_Previews: PreviewProvider {
    static var previews: some View {
        QRCodeView(qrCodeType: .bitcoin("bitcoinqrcode"))
        QRCodeView(qrCodeType: .lightning("lightingqrcode"))
    }
}

**Comment by reez** [comment]
Hi thanks for the heads up!

Yup that `QRCodeView` comes from [BitcoinUI](https://github.com/reez/BitcoinUI) which is a library of mine. The background/context is that when iOS 17 came out I removed `PreviewProvider` and replaced with the new `#Preview`. But now I've logged an Issue on `BitcoinUI` to support `PreviewProvider` again : https://github.com/reez/BitcoinUI/issues/86

If you have any other thoughts/details feel free to let me know, otherwise thanks for that feedback and opening a great Issue!
